pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotInstanceRestletResource.java

```diff
@@ -390,4 +391,33 @@ public SuccessResponse updateBrokerResource(
           Response.Status.INTERNAL_SERVER_ERROR, e);
     }
   }
+
+  @POST
```

Comment:
Since this is just reading some information without applying any change to the cluster, should we make it a POST request of GET?

Reply:
+1

Reply:
Agree, will make the changes.